libs/fills/tsconfig.spec.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node", "@testing-library/jest-dom"]
+  },
+  "include": [
+    "**/*.test.ts",
+    "**/*.spec.ts",
+    "**/*.test.tsx",
+    "**/*.spec.tsx",
+    "**/*.test.js",
+    "**/*.spec.js",
+    "**/*.test.jsx",
+    "**/*.spec.jsx",
+    "**/*.d.ts"
```

Comment:
would be great to exclude all the stories.* files from tsconfig.lib.json as in https://github.com/vegaprotocol/frontend-monorepo/pull/610/files#diff-c4c808cbef23c19ee8c600f01c29a81bab02a57a053fe1bcebcee1ccd173aec7